Frontend/src/Components/BatteryCells/DataPack.js

```diff
@@ -0,0 +1,25 @@
+import { Box, Text, VStack, Flex, Spacer } from "@chakra-ui/react";
+
+export default function DataPack(props) {
+    return (
+        <VStack
+            borderWidth={1}
+            borderColor='black'
+            spacing={0}
+            bg={props.bg}
+        >
+            <Flex w='95%'>
+                <Text fontSize='xs'>
+                    {props.dataTitle}:
+                </Text>
+                <Spacer />
+                <Text fontSize='xs'>
```

Comment:
If the fontSize can be increased from xs to sm (or some value in-between) after removing the nominal range boxes and resizing the components to fit the data view, that would be ideal. If not, fontSize can stay as xs.